archinstall/lib/exceptions.py

```diff
@@ -27,33 +24,13 @@ def __init__(self, message :str, exit_code :Optional[int] = None, worker :Option
 		self.worker = worker
 
 
-class PermissionError(BaseException):
-	pass
-
-
-class ProfileNotFound(BaseException):
-	pass
-
-
 class HardwareIncompatibilityError(BaseException):
 	pass
 
 
-class UserError(BaseException):
-	pass
-
-
 class ServiceException(BaseException):
 	pass
 
 
 class PackageError(BaseException):
 	pass
-
-
-class TranslationError(BaseException):
-	pass
-
-
-class Deprecated(BaseException):
```

Comment:
Could we keep this one? It feels like this might have been removed by an automatic tool because it's currently not used anywhwre?

Reply:
I searched the codebase and couldn't find any usages, bu I'm happy to keep it

Reply:
It's used whenever we deprecate something, so it's used from time to time - and people can use it in custom scripts to detect deprecations easier :)